src/styles/index.scss

```diff
@@ -1,3 +1,300 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
 }
+
+/* #region HEADER */
+
+@mixin flex-center($justify: space-between, $align: center) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+.header {
+  @include flex-center;
+
+  height: 60px;
+  padding: 0 50px;
+
+  &__nav {
+    padding: 0;
+
+    // Навігаційна панель
+    &-list {
+      list-style: none;
+      padding: 0;
+      margin: 0;
+
+      @include flex-center(flex-start);
+    }
+
+    &-item {
+      margin-right: $nav-item-margin;
+      font-size: $font-size-small;
+      font-weight: $font-weight-medium;
+      text-transform: uppercase;
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      &__link {
+        color: #060b35;
+        text-decoration: none;
+        display: block;
+        line-height: $line-height-nav;
+
+        &:hover,
+        &:active {
+          color: $primary-color;
+        }
+
+        &:active {
+          text-decoration: underline;
+        }
+
+        &.is-active {
+          position: relative;
+          color: $primary-color;
+
+          &::after {
+            content: '';
+            display: block;
+            width: 100%;
+            height: 4px;
+            background-color: $primary-color;
+            border-radius: 2px;
+            position: absolute;
+            bottom: 0;
+            left: 0;
+          }
+        }
+      }
+    }
+  }
+
+  &__logo-link,
+  &__logo-link img {
+    width: $logo-size;
+    height: auto;
+    display: block;
+  }
+}
+
+/* #endregion HEADER */
+
+/* #region CATALOG */
+.catalog {
+  display: grid;
+  width: 944px;
+
+  grid-template-columns: repeat(4, 200px); /* 4 колонки по 200px */
```

Comment:
Avoid using fixed values for the grid template columns. It can lead to issues with responsiveness. Instead, consider using relative units like fr or auto-fill combined with minmax for more flexible and responsive layouts.